src/Compilers/CSharp/Test/Semantic/Semantics/NameOfTests.cs

```diff
@@ -1168,5 +1168,66 @@ public void NameofTypeParameterInParameterInitializer()
 }";
             var compilation = CreateCompilationWithMscorlib45(source).VerifyDiagnostics();
         }
+
+        [Fact, WorkItem(10467, "https://github.com/dotnet/roslyn/issues/10467")]
+        public void NameofFixedBuffer()
```

Comment:
Master is merged to the future on a daily basis. Rather than waiting for a merge from future to master, the test can be properly marked in the future, once it gets there.